Resources/Audio/_Goobstation/Voice/Human/attributions.yml

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2024 Piras314 <p1r4s@proton.me>
+# SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>
+# SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+# SPDX-FileCopyrightText: 2025 SX-7 <sn1.test.preria.2002@gmail.com>
+# SPDX-FileCopyrightText: 2025 loltart <lo1tartyt@gmail.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+- files: ["gulp.ogg"]
+  license: "CC0-1.0"
```

Comment:
The sounds are not CC0, they're licensed under the Custom [license pixabay uses](https://pixabay.com/service/terms/)
Which says `CC0 Content on the Service is any content which lists a "Published date" prior to January 9, 2019.`, which doesn't apply to either of the sounds added here